### PR TITLE
Created an API to get the last known status for a station

### PR DESCRIPTION
**Pull request by bsm5**
My tests were run on postman using localhost:3001.  I believe I covered all my basis but please let me know if you catch anything not working as it should.

Note - I'll be removing the console.logs before I merge.  I used them for testing and figured I'd leave them in to help others test.

### CODE REVIEW

File: routes/api.js [hunk 1]
```diff
@@ -81,5 +82,91 @@ router.post('/station-heartbeat', function(req, res) {
...
+                res.set({
+                  'Content-Type': 'text/plain',
+                  'station-state': 'disabled',
+                });
+                console.log('Did we add something to the cache?');
+                res.sendStatus(200);
+              });
+            }else{
+              db.logEvent('station-status request', 'Station: ' + sid + ' has an unknown state and is unable to be add to cache').then(function(){
+                res.sendStatus(403);
```
Comment (mongolsamurai) [inline routes/api.js]: This looks like it's an error handler?
If so, this should return a `500` error.

File: routes/api.js [hunk 2]
```diff
@@ -81,5 +82,91 @@ router.post('/station-heartbeat', function(req, res) {
...
+      }else{
+        db.logEvent('station-status request', 'Station: ' + sid + ' cannot be found.').then(function(){
+        console.log('Empty Results');
+        res.sendStatus(403);
+        });
+      }
+    }).catch(err => {
+      db.logEvent('station-status request', err).then(function(){
+      console.log(err);
+      res.sendStatus(403);
```
Comment (mongolsamurai) [inline routes/api.js]: Also should be a 500 error

File: routes/api.js [hunk 3]
```diff
@@ -81,5 +82,91 @@ router.post('/station-heartbeat', function(req, res) {
...
+      });
+    });
+  }else{
+    console.log('No sid or Cert');
+    res.sendStatus(403);
```
Comment (mongolsamurai) [inline routes/api.js]: This should probably be a 400 error (missing header == malformed request)

**Comment by bsm5** [comment]
@mongolsamurai All requested changes should be fixed now.

**Pull request merged**